tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -57,6 +57,7 @@ const childrenCards = () => $$('.right-pane .card.children');
 const contactCardTitle = () => $('.inbox .content-pane .material .body .action-header');
 const contactInfoName = () => $('h2[test-id="contact-name"]');
 const contactMedicID = () => $('#contact_summary .cell.patient_id > div > p');
+const contactPhone = () => $('#contact_summary .cell.phone > div > p');
```

Comment:
Please move this selector and its method to the file located in the `standard` folder [/standard/contacts/contacts.wdio.page.js](https://github.com/medic/cht-core/blob/7b897aa290baa74f0d80e7b2009bd71564c71e10/tests/page-objects/standard/contacts/contacts.wdio.page.js), because it is not a valid one for the default config.

### Default config:

To work in the `default` config it will need to be something like this: `#contact_summary .cell.person\.field\.phone > div > p` but this one doesn't work for the `standard` config 😅 

![image](https://github.com/medic/cht-core/assets/94494491/f46a3640-874f-47eb-b2a4-62dbfca2345c)

### Standard config:
 
![image](https://github.com/medic/cht-core/assets/94494491/a00c2f00-de45-4804-b016-bfdc147efd92)

Reply:
Done! Thanks for noticing!